pkg/base/capacityplanner/common.go

```diff
@@ -39,6 +39,15 @@ func AlignSizeByPE(size int64) int64 {
 	return size + alignement
 }
 
+// AlignSizeByMB make size aligned with MB
+func AlignSizeByMB(size int64) int64 {
```

Comment:
please add unit-test in common_test.go for this function.

Reply:
added